fixtures/capSetConfig.yaml

```diff
@@ -0,0 +1,15 @@
+capabilitiesToBeDropped:
```

Comment:
Is this missing one cap on purpose? If so please document that behavior, if not please add it.